subworkflows/local/vcf_sites_extract_bcftools/main.nf

```diff
@@ -0,0 +1,71 @@
+include { BCFTOOLS_VIEW as VIEW_VCF_SNPS         } from '../../../modules/nf-core/bcftools/view/main.nf'
+include { BCFTOOLS_VIEW as VIEW_VCF_SITES        } from '../../../modules/nf-core/bcftools/view/main.nf'
+include { BCFTOOLS_INDEX                         } from '../../../modules/nf-core/bcftools/index/main.nf'
+include { BCFTOOLS_INDEX as BCFTOOLS_INDEX_2     } from '../../../modules/nf-core/bcftools/index/main.nf'
```

Comment:
We should stick to one method to rename modules when we do multiple import.
Which one would you prefer INDEX or BCFTOOLS_INDEX ?

Reply:
` BCFTOOLS_INDEX` is more informative

Reply:
Okay so we might need to change it in the rest of the pipeline.
I'll add it to the issues ;)